admin/company/settings/domains.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Domain management in Docker Admin
+keywords: domains, SCIM, SSO, Docker Admin
+title: Domain management
+---
+
+{% include admin-early-access.md %}
+
+Use domain management to manage your domains for Single Sign-On and SCIM.
+
+## Add and verify a domain
+
+1. Sign in to [Docker Admin](https://admin.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. In the left navigation, select your company in the drop-down menu.
+3. Select **Domain management**.
+4. Select **Add a domain** and continue with the on-screen instructions to add the TXT Record Value to your domain name system (DNS).
```

Comment:
For your radar: this is the section mentioned in the Accounts meeting that may not be clear enough to all audiences what domain they should use for the DNS record. I opened a JIRA ticket to update that on the other page. Something to consider with maintaining these similar admin pages 😄